chef/cookbooks/rabbitmq/recipes/rabbit.rb

```diff
@@ -81,7 +81,10 @@
   only_if only_if_command if ha_enabled
 end
 
-if node[:rabbitmq][:trove][:enabled]
```

Comment:
@mjura @vuntz Sorry to answer that late

The attribute part of the default template (and got added with a migration as part of https://github.com/crowbar/barclamp-rabbitmq/blob/master/chef/data_bags/crowbar/migrate/rabbitmq/011_trove.rb).  And yes to be able to use trove you need to manually set it to "true". IIRC we agreed to do it that somewhat awkward way because of the security implications that enabling access to the MessageQueue via the public network has. IMO that is still an issue we need to solve before we can really support trove.